src/Silverback.Integration/Messaging/Sequences/SequencerProducerBehavior.cs

```
@@ -47,18 +51,21 @@ public async Task HandleAsync(ProducerPipelineContext context, ProducerBehaviorH
                 if (!sequenceWriter.CanHandle(context.Envelope))
                     continue;
 
+                if (cancellationToken.IsCancellationRequested)
```

Throw instead of silently aborting here too?